README.md

```diff
@@ -142,11 +142,13 @@ The following is a brief explanation of each available command.
   CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze // 30 minutes
   ```
 
+  Note: CONTAINER_TIMEOUT_SECONDS is not currently honored in the CLI.
```

Comment:
@jrafanie I was able to get it to work like this:

```console
~/git_repos/manageiq (master) 🐖  CONTAINER_TIMEOUT_SECONDS=30 codeclimate analyze
Starting analysis
Running csslint: Done!
Running duplication: Done!
error: (CC::Analyzer::Engine::EngineTimeout) engine duplication:stable ran for 30 seconds and was killed
```

Can you run `cat $(which codeclimate)` and share the output? You might be using an outdated wrapper